### ISSUE TITLE
improve a11y of video transcripts

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#6984]
### Observed behavior

follow-up from https://github.com/learningequality/kolibri/pull/5631#issuecomment-551367866

when the user is navigating up and down the transcript **while video is playing**, video player bar with all the rest of controls disappears, not only visually, but from the TAB order too. This results in user being unable to navigate back and pause/stop or control the playback in any other way. Happens with both positions of the transcript (sidebar and below)

([screencast here, too big for GH attachment](https://drive.google.com/open?id=1YaGAc0lM4TIQsWXQmcYo89I1hWos9cWE))

Also when the video is long, and the list of captions becomes interminable.

### Expected behavior

* in keyboard modality, we keep the control bar visible regardless
* changing the roles of the items in the transcript such that it's traversalable by arrow keys and perhaps <kbd>Tab</kbd> jumps between `Beginning of transcript` and `End of transcript`
* add support for  <kbd>Home</kbd> and <kbd>End</kbd>

### User-facing consequences

reduced accessibility of transcripts

### Context

0.13

**Comment by chrislun16** [comment]
Is this still open? Can I be assigned to this? We're not sure how to approach it yet.

For a school assignment, my partner and I have to contribute to an open source project and choose a few tasks. So I chose this repo and I wanted to find a few tasks like this one that I can be assigned to. I wanted to be assigned before someone else comes and completes the task.

**Comment by pcenov** [comment]
@radinamatic this is not extant in 0.15.1 where the video controls do disappear as mentioned above but reappear once tab is pressed or the user hovers the mouse over the player.

**Comment by rtibbles** [comment]
Closing as no longer extant.
